capstone/src/test/java/com/google/univiz/api/SearchResourceImplTest.java

```diff
@@ -0,0 +1,36 @@
+package com.google.univiz.api;
+
+import static com.google.common.truth.Truth.assertThat;
+import static org.mockito.Mockito.when;
+
+import java.util.ArrayList;
+import java.util.List;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+import org.mockito.Mock;
+import org.mockito.junit.MockitoJUnit;
+import org.mockito.junit.MockitoRule;
+
+@RunWith(JUnit4.class)
+public final class SearchResourceImplTest {
+  @Mock private SuggestionDataApi mockSuggestionApi;
+
+  @Rule public MockitoRule mockitoRule = MockitoJUnit.rule();
```

Comment:
nit - +final

Reply:
added it to the SuggestionDataApi

Reply:
hmm tried adding it and it broke the test...

Reply:
This comment is about line 20:
```suggestion
  @Rule public final MockitoRule mockitoRule = MockitoJUnit.rule();
```

Reply:
ah that makes a lot more sense-thanks!